common/src/main/java/com/wynntils/models/players/hades/objects/HadesUser.java

```diff
@@ -30,6 +30,25 @@ public HadesUser(HSPacketUpdateMutual packet) {
         this.updateFromPacket(packet);
     }
 
+    // Dummy constructor for previews
+    public HadesUser(String name, CappedValue health, CappedValue mana) {
+        this.uuid = UUID.randomUUID();
```

Comment:
This feels wrong. Can't we feed it the dummy values from outside?

Also, UUID.randomUUID() for preview seems scary. You could get anything, including NSFW skins. Better off hard-coding Alex/Steve.